code/game/objects/items/emags.dm

```diff
@@ -73,6 +94,25 @@
 			return FALSE
 	return TRUE
 
+/// What happens if a crewmember uses this card
+/obj/item/card/emag/proc/crew_action(mob/user)
+	if(!HAS_TRAIT(user, TRAIT_EMAG_USER))
+		crew_uses += 1
+	if(!prob(crew_uses * 25))
```

Comment:
This should be before you increment the emag. This isn't so much as a means to still use it as crew at least one time, but as a way to create uncertainty about the emag user.